monors.py

```diff
@@ -141,6 +141,15 @@ def has_merge_command (self):
 
         return False
 
+    def has_merge_command (self):
+        return self.has_command("merge")
+
+    def has_squash_command (self):
+        return self.has_command("squash")
+
+    def has_rebase_command (self):
+        return self.has_command("rebase")
```

Comment:
I'm wondering whether we should call the new commands `squash and merge` and `rebase and merge` to be more aligned with what they're called in the GitHub merge button?